frontend/src/routes/_dashboard.tsx

```diff
@@ -86,14 +93,14 @@ function Header() {
         <Link href="/">
           <Logo className="w-10 flex-none h-10 mr-8" />
         </Link>
-        <div className="flex  w-full items-center">
+        <div className="md:flex hidden  w-full items-center">
           <Menubar className="border-none w-fit text-black bg-primary">
             <MenubarMenu>
               <MenubarTrigger className="flex  justify-center text-sm items-center gap-1">
                 Create
                 <ChevronsUpDown className="w-4" />
               </MenubarTrigger>
-              <MenubarContent className=" min-w-6">
+              <MenubarContent className=" border border-border min-w-6">
```

Comment:
What is `border border` ?

Reply:
border border-border i'm saying here to put a border then with a color variable defined to all borders in global.css named border.